File: Ser2025/Company/Company.Web/Modules/ServerTypes/Company/PhoneNumberTypeForm.ts
import { PrefixedContext, IntegerEditor, StringEditor, initFormType } from '@serenity-is/corelib';

export interface PhoneNumberTypeForm {
    Name: StringEditor;
    IsActive: IntegerEditor;
    TenantId: IntegerEditor;
}

export class PhoneNumberTypeForm extends PrefixedContext {
    static readonly formKey = 'Company.PhoneNumberType';
    private static init: boolean;
    
    constructor(prefix: string) {
        super(prefix);
        if (!PhoneNumberTypeForm.init)  {
            PhoneNumberTypeForm.init = true;
            
            var w0 = IntegerEditor;
            var w1 = StringEditor;

            initFormType(PhoneNumberTypeForm, [
            'Name', w1,
            'IsActive', w0,
            'TenantId', w0,
            ]);
        }
    }
}

File: Ser2023/Company/Company.Web/Modules/Company/PhoneNumberType/PhoneNumberTypeForm.cs
using Serenity.ComponentModel;
using System;

namespace Company.Company.Forms;

[FormScript("Company.PhoneNumberType")]
[BasedOnRow(typeof(PhoneNumberTypeRow), CheckNames = true)]
public class PhoneNumberTypeForm
{
    public string Name { get; set; }
    //public Guid Rowguid { get; set; }
    //public short IsActive { get; set; }
    //public int TenantId { get; set; }
}